### PR TITLE
review & approval correction

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Per @mcampo and @kuntald email: "That documentation is incorrect. We do not have a "publisher" role. Exchange only has 3 asset roles; "admin", "contributor" and "viewer". Contributors can create the draft AND publish it."

### CODE REVIEW

**Review by mcampo — COMMENTED** [review]
The title of the section is `What is the expected review and approval workflow?`, and actually we do not have a "review and approval workflow".

What we have is a draft state where that the contributor can use to stage changes and possibly get feedback from others. The same user can publish those changes. We do not have a way to separate the responsibility of publishing the content.

Also, note that custom roles cannot be assigned Exchange permissions. i.e. If I create a custom role "My Role", I cannot assign "Exchange contributor" permission to it.

I see a few alternatives, would like to get @kuntald opinion on this:
 1. change the title of the section so it does not imply there is a "review and approve workflow", and explain the draft / publish flow.
 2. keep the section title (because maybe that's what the user is actually searching for), explicitly mention that there is no separation between writer and approver, and explain the draft / publish flow.
 3. remove this section altogether so we don't confuse users

**Comment by wwarner-salesforce** [comment]
What permissions do custom roles have? Do all custom roles have the same permissions as the Contributor asset role?

**Comment by mcampo** [comment]
you can assign roles from other products, but not Exchange
image

Try it out yourself to get an idea of what can be done.

**Comment by wwarner-salesforce** [comment]
Proposed new draft:

== What is the expected workflow for reviewing, approving, and publishing drafts?

There are three asset roles in Exchange: Admin, Contributor, and Viewer. You
can also create custom roles in Anypoint Platform > Access Management > Roles.
Organizations can create two custom roles for "contributor" and "publisher."

Exchange provides a drafts feature. A user who has the custom "contributor"
role can create drafts and share the URLs with a user who has the custom
"publisher" role. The user who has the custom "publisher" role can review,
approve, and publish these pages. Note that all custom roles have the same
permissions, including the permission to publish drafts, so the separation of
roles is not enforced by Exchange.

Content in the draft state is hidden until the draft is published.

**Comment by wwarner-salesforce** [comment]
Or with no mention of roles:

== What is the expected workflow for reviewing, approving, and publishing drafts?

Exchange provides a drafts feature. Content in the draft state is hidden until the draft is published.

One user can create drafts and share the URLs with a second user, and the second user can review, approve, and publish these pages.

All contributors have the same permissions, including the permission to publish drafts. The separation of duties is not enforced by Exchange and must be observed by the users.

**Comment by mcampo** [comment]
I like the second one better, but let's not say that the second user can "approve" because there is no actual action for that.

**Comment by wwarner-salesforce** [comment]
Good point.

== What is the expected workflow for reviewing and publishing drafts?

Exchange provides a drafts feature. Content in the draft state is hidden until the draft is published.

One user can create drafts and share the URLs with a second user, and the second user can review and publish these pages.

All contributors have the same permissions, including the permission to publish drafts. The separation of duties is not enforced by Exchange and must be observed by the users.

**Comment by mcampo** [comment]
Looks good to me

**Comment by wwarner-salesforce** [comment]
Sounds good. Based on discussion with Kuntal at the beginning of the meeting, I'll merge this, and he can review it when he gets time, and then send me any further changes.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**
